apps/asap-server/src/controllers/teams.ts

```diff
@@ -39,40 +39,48 @@ const transformUser = (users: CMSUser[], teamId: string): TeamMember[] =>
   }));
 
 const fetchUsers = async (id: string, client: Got): Promise<CMSUser[]> => {
-  try {
-    const { items } = await client
+  const [, res] = await Intercept(
+    client
       .get('users', {
         searchParams: {
           $filter: `data/teams/iv/id eq '${id}'`,
         },
       })
-      .json();
+      .json() as Promise<{ total: number; items: CMSUser[] }>,
+  );
 
-    return items;
-  } catch (err) {
-    return [];
-  }
+  return res ? res.items : [];
 };
 
 export default class Teams {
-  cms: CMS;
-
   teams: Squidex<CMSTeam>;
 
   users: Squidex<CMSUser>;
 
   constructor() {
-    this.cms = new CMS();
     this.teams = new Squidex('teams');
     this.users = new Squidex('users');
   }
 
   async fetch(options: {
     take: number;
     skip: number;
+    search: string;
   }): Promise<ListTeamResponse> {
+    const { search, ...opts } = options;
     const { total, items: teams } = await this.teams.fetch({
-      ...options,
+      ...opts,
+      ...(search
+        ? {
+            fullText: search,
+            filter: {
+              or: [
+                { path: 'data.displayName.iv', op: 'contains', value: search },
```

Comment:
To make the search more helpful we should tokenize `search` and filter on multiple strings.

Reply:
You mean, splitting by whitespace and applying the same filters on all of those tokens?